hack/make/.detect-daemon-osarch

```diff
@@ -62,6 +62,9 @@ case "$PACKAGE_ARCH" in
 				;;
 		esac
 		;;
+	aarch64)
+		DOCKERFILE="Dockerfile.$PACKAGE_ARCH"
+		;;
```

Comment:
Is this here simply to skip `TEST_IMAGE_NAMESPACE`?  It seems like it'd be more appropriate to put an `if` in the `*` block here, wouldn't it?  Don't we still need to set it to _something_, or is the empty value handled OK elsewhere?

Reply:
The reason to skip `TEST_IMAGE_NAMESPACE` here is we've already switched to the multi-arch images on AArch64, so we don't need a name space like `aarch64/` any more order to pass the `test-integration`. I think we will not need `TEST_IMAGE_NAMESPACE` if we switch all the legacy images to  multi-arch ones in the future. But now let's it be and I add a `if` in the `*` block to handle AArch64 specific.